### ISSUE TITLE
Setup background of game

### ISSUE THREAD

**Issue by BinojBI** [BinojBI/Survivor#4]

**Comment by TDhananjaya** [comment]
komada hadenna ona island eka

**Comment by BinojBI** [comment]
limbo game eka balapn, godak durata hadenne e widiyata. eke design ekak
oona ikmanatama

**Comment by TDhananjaya** [comment]
3d neweida?

**Comment by BinojBI** [comment]
ne.  3d oona ne

**Comment by TDhananjaya** [comment]
unity wala tina package ganne nathuwada hadanne

**Comment by BinojBI** [comment]
Yes you can use software for modeling purpose finally outcome should be only the model. Its not essential to to be 3d. you can design them 2d also using photoshop.

**Comment by BinojBI** [comment]
Hello there,

I just signed the petition "Prime Minister of Sri Lanka: Pass and Enact the
Animal Welfare Bill of Sri Lanka to replace current law from 1907." and
wanted to see if you could help by adding your name.

Our goal is to reach 59,318 signatures and we need more support. You can
read more and sign the petition here:

http://chng.it/TwZ9NgD9P4

Thanks!
Binoj

**Comment by BinojBI** [comment]
Hello there,

I just signed the petition "President of Srilanka : Rishard Badurdeen
should be sacked from his post & questioned on Easter Sunday attacks." and
wanted to see if you could help by adding your name.

Our goal is to reach 5,000 signatures and we need more support. You can
read more and sign the petition here:

http://chng.it/KHsttMpvGp

Thanks!
Binoj
